### ISSUE TITLE
Make shorty url support focal point

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#17965]
We need to finish what was started in 5.2.5 and make the ShortyServlet respect focal point, e.g.
e.g.

http://localhost:8080/dA/c56e5030fc88/200cw/200ch/.3,.5fp/100w

**Comment by wezell** [comment]
Needs testing - we need to make sure we don't don't break the other shorty options AND that we don't break the webp filter

**Comment by wezell** [comment]
PR: #17979

**Comment by bryanboza** [comment]
Fixed, tested on release-5.2.7 // Postgres // FF

Tested the crop and focal point filer using shortly ids

**Comment by wezell** [comment]
Here are the parameters that are accepted by the short image path:

```    
final int      width   =  `100w`
    final int      height  = `100h`
    final int      quality  = this.getQuality(lowerUri, 0);
    final Optional<FocalPoint> focalPoint = this.getFocalPoint(lowerUri);
    final int      cropWidth  = this.cropWidth(lowerUri);
    final int      cropHeight  = this.cropHeight(lowerUri);
    final boolean  jpeg    = lowerUri.contains(JPEG);
    final boolean  jpegp   = jpeg && lowerUri.contains(JPEGP);
    final boolean  webp    = lowerUri.contains(WEBP);
```
